doc/ref/copyright.xml

```diff
@@ -66,28 +66,20 @@ make.
 In addition to the general copyright for &GAP; set forth above,
 the following terms apply to the versions of &GAP; for Windows.
 <P/>
-The executable of &GAP; for Windows that we distribute was compiled with
+&GAP; for Windows is packaged with several packages from
+<Package>Cygwin</Package> (<URL>https://cygwin.com</URL>).
+Files from <Package>Cygwin</Package> are provided under their respective
+licenses (included).
+The &GAP; for Windows executables that we distribute are released under
+the terms of the GNU General Public License (GPL).
+These &GAP; executables were compiled with
 the <C>gcc</C> compiler supplied with <Package>Cygwin</Package> installation
-(<URL>https://cygwin.com/</URL>).
 <P/>
 The GNU C compiler is
 <P/>
 <E>Copyright &copyright; 2010 Free Software Foundation, Inc.</E>
 <P/>
-under the terms of the GNU General Public License (GPL).
-<P/>
-The <Package>Cygwin</Package> API library is also covered by the GNU GPL. 
-The executable we provide is linked against this library (and in the process 
-includes GPL'd <Package>Cygwin</Package> glue code). This means that the 
-executable falls under the GPL too, which it does anyhow.
-<P/>
-The <F>cyggcc_s-1.dll</F>, <F>cygncurses-10.dll</F>, <F>cygncursesw-10.dll</F>,
-<F>cygpanel-10.dll</F>, <F>cygpopt-0.dll</F>, <F>cygreadline7.dll</F>,
-<F>cygstart.exe</F>, <F>cygwin1.dll</F>, <F>libW11.dll</F>, <F>mintty.exe</F> 
-and <F>regtool.exe</F> are taken unmodified from the 
-<Package>Cygwin</Package> distribution. They are copyright by RedHat Software
-and released under the GPL. For more information on <Package>Cygwin</Package>,
-see <URL>https://www.cygwin.com</URL>.
+under the terms of the GNU GPL.
```

Comment:
Also say which version?

Reply:
I'm pretty sure it's version 3, so I've changed it to that, but I'm not 100% sure (perhaps it is v3 or later?). Since we're not including any of the gcc source code (as far as I'm aware) I don't think we actually _need_ to be talking about GCC here. That said I'm happy to mention it.

Reply:
Ah, ok